src/goto-symex/goto_symex.h

```diff
@@ -56,6 +56,8 @@ class goto_symext
     path_storaget &path_storage)
     : should_pause_symex(false),
       options(options),
+      max_depth(options.get_unsigned_int_option("depth")),
+      doing_path_exploration(options.is_set("paths")),
```

Comment:
Would you argue that this fixes #2050? If so, please add it to the commit message.